src/jit/unwindamd64.cpp

```diff
@@ -19,6 +19,44 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
 #if defined(_TARGET_AMD64_)
 
+int Compiler::mapRegNumToDwarfReg(regNumber reg)
+{
+    assert(reg >= 0 && reg <= REG_XMM15);
+    int dwarfReg = reg;
+
+    switch (reg) {
+    case REG_RAX: dwarfReg = 0; break;
+    case REG_RCX: dwarfReg = 2; break;
+    case REG_RDX: dwarfReg = 1; break;
+    case REG_RBX: dwarfReg = 3; break;
+    case REG_RSP: dwarfReg = 7; break;
+    case REG_RBP: dwarfReg = 6; break;
+    case REG_RSI: dwarfReg = 4; break;
+    case REG_RDI: dwarfReg = 5; break;
+    default:
+        if (reg >= REG_XMM0) {
+            dwarfReg++; // shift by 1 due to dwarf_reg_ip
```

Comment:
I validated this CFI assembly directive. Dwarf XMM0 starts with 16 instead of REG_XMM0 (15). Note actually since all xmm are volatile in Unix ABI, we don't save them at all so this part is actually not used, but leave it for consistency.